Cargo.toml

```diff
@@ -6,7 +6,7 @@ license = "Apache-2.0"
 
 [dependencies]
 bech32 = "0.5.0"
-secp256k1 = "0.10"
-bitcoin = "0.13"
+secp256k1 = "0.11"
+bitcoin = "0.14"
```

Comment:
It seems `bitcoin 0.14` wasn't published yet.

Reply:
From IRC: `bitcoin 0.14.0` couldn't be published because of a git dependency. So basically this means I will wait till this is resolved.